script/hassfest/manifest.py

```diff
@@ -51,11 +51,20 @@ def verify_uppercase(value: str):
     return value
 
 
+def verify_version(value: str):
+    """Verify the version."""
+    version = AwesomeVersion(value)
+    if version.strategy == AwesomeVersionStrategy.UNKNOWN:
```

Comment:
Instead of comparing to unknown strategy, should we instead compare to exact strategy that we support?

Reply:
Sure :+1: